package.json

```diff
@@ -197,7 +197,14 @@
         "setBabelEnv": false,
         "babel": {
           "presets": [
-            "node5"
+            [
+              "env",
+              {
+                "targets": {
+                  "node": "current"
```

Comment:
Haha. 'current' right now is 9.4.0. Atom won't be able to read any of it if you transpile to that.

Change this to `"node": "7.4.0".

Reply:
That's not what that actually means. As [the documentation](https://www.npmjs.com/package/babel-preset-env#targetsnode) states `current` means "use the version of Node.js that this is _currently_ running under.

Reply:
Updated the PR description to link there as well so this was a bit clearer 😉.

Reply:
Hmm, interesting. I did not realize that. Great call then. That will keep it evergreen. :🤸‍♂️